serverless.yml

```
@@ -0,0 +1,36 @@
+service: sosrs
+frameworkVersion: '3'
+
+provider:
+  name: aws
+  region: us-east-1
+  runtime: nodejs18.x
+  logRetentionInDays: 7
+  versionFunctions: false
+
+  httpApi:
+    cors: true
+    metrics: true
+
+# Adapted from:
+#   https://www.prisma.io/docs/orm/prisma-client/deployment/serverless/deploy-to-aws-lambda#deploy-only-the-required-files
+package:
+  patterns:
+    - '!node_modules/.prisma/client/libquery_engine-*'
+    - 'node_modules/.prisma/client/libquery_engine-rhel-*'
+    - '!node_modules/prisma/libquery_engine-*'
+    - 'node_modules/@prisma/engines/**'
+    - '!node_modules/.cache/prisma/**'
+
+functions:
+  backend:
+    timeout: 30
+    memorySize: 256
+    handler: dist/lambda.handler
+    events:
+      - httpApi:
+          method: any
+          path: '/{proxy+}'
```

Normalmente se aplica um agendamento assim para mitigar a questão que citou sobre o cold-start da Lambda.

```suggestion
    events:
      - httpApi:
          method: any
          path: '/{proxy+}'
      - schedule: rate(2 minutes)
```